vllm/config.py

```diff
@@ -2149,6 +2151,11 @@ class CompilationConfig(BaseModel):
     level: int = 0
     backend: str = ""
     custom_ops: List[str] = Field(default_factory=list)
+    splitting_ops: List[str] = Field(default_factory=lambda: [
+        "vllm.unified_flash_attention",
+        "vllm.unified_flash_infer",
+        "vllm.unified_v1_flash_attention",
+    ])
```

Comment:
What are these default values for?

Reply:
they are the default splitting ops we know inductor cannot do any optimization, and they are good candidates to split the model.